plugins/mission/mocks/mission_mock.h

```diff
@@ -8,13 +8,14 @@
 namespace dronecore {
 namespace testing {
 
-typedef std::function<void(Mission::Result)> result_callback_t;
+typedef std::function<void(Mission::Result)> mission_result_callback_t;
```

Comment:
Can't you get this typedef from somwhere else? Or is it specific to testing?

Reply:
You're right. I did not want to take `Mission::result_callback_t`, but for wrong reasons :-).